x-pack/plugin/autoscaling/src/test/java/org/elasticsearch/xpack/autoscaling/storage/ReactiveStorageDeciderDecisionTests.java

```diff
@@ -405,6 +449,18 @@ private long numAllocatableSubjectShards() {
             .count();
     }
 
+    private SortedSet<ShardId> allocatableShards() {
```

Comment:
I think this method and `numAllocatableSubjectShards` can be collapsed into one, returning the `ShardsSize` to use.